PLUB/Sources/Views/Home/Cell/RegisterInterest/RegisterInterestDetailTableViewCell.swift

```diff
@@ -111,9 +118,13 @@ extension RegisterInterestDetailTableViewCell: UICollectionViewDelegate, UIColle
   
   func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
     guard let cell = collectionView.cellForItem(at: indexPath) as? InterestTypeCollectionViewCell,
-    let mainIndexPath = self.indexPath else {
+    let mainIndexPath = self.indexPath, let selectEnabled = selectEnabled else {
       return
     }
+    let currentSeleted = subCategories[indexPath.row].isSelected
+    
+    guard selectEnabled || (currentSeleted && !selectEnabled) else { return }
```

Comment:
이렇게 줄일 수 있습니다!
```suggestion
    guard selectEnabled || currentSeleted else { return }
```
왜냐하면, `selectedEnabled`가 `true`일 경우 뒤의 조건식은 확인하지 않고 넘어가겠지만, 만약 `selectedEnabled`가 `false`일 경우에는 다음 조건식을 확인하게 되는데, 이 때 `!selectedEnabled`는 자연스레 `true` 가 되므로, 제거해도 무방하기 때문입니다.

Reply:
헛 맞네요!! 덕분에 조건식이 너무 깔끔해졌네요 승현님 감사합니다!